### PR TITLE
OSSMDOC-627: Remove duplicate ID

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.6 - 4.11

Issue: https://issues.redhat.com/browse/OSSMDOC-627

Link to docs preview: http://file.bos.redhat.com/tokeefe/OSSMDOC-627/service_mesh/v1x/ossm-security.html#ossm-cert-manage_ossm-security-v1x

Additional information:
Peer review: jstickler

### CODE REVIEW

**Comment by rh-tokeefe** [comment]
@JStickler this is ready for review.

**Comment by JStickler** [comment]
@rh-tokeefe I can see that you've updated the PR, but I'm still seeing the same v2 content in the preview.  Could you regenerate and sync the preview again please?
Preview

File: modules/ossm-security-mtls-1x.adoc [hunk 1]
```diff
@@ -70,7 +70,7 @@ If your environment has specific requirements for encrypted traffic in your serv
 
 [source,yaml]
 ----
-apiVersion: maistra.io/v2
+apiVersion: maistra.io/v1
```
Comment (JStickler) [inline modules/ossm-security-mtls-1x.adoc:73]: The current YAML example is against the OSSM v2 SMCP schema.  I had to dig around in the archives to find the proper YAML for a v1 SCMP.    Here it is.

[source,yaml]
----
apiVersion: maistra.io/v1
kind: ServiceMeshControlPlane
spec:
  istio:
    global:
      tls:
        minProtocolVersion: TLSv1_2
        maxProtocolVersion: TLSv1_3
----

Comment (rh-tokeefe) [reply modules/ossm-security-mtls-1x.adoc:73]: @JStickler thanks for digging into the old v1 SMCP YAML file. I've updated the source and the preview build is available.

**Review by JStickler — APPROVED** [review]
LGTM

**Pull request merged**